lib/Conversion/TorchToTosa/TosaLegalizeCommon.cpp

```diff
@@ -27,6 +28,370 @@
 namespace mlir {
 namespace tosa {
 
+using namespace mlir::torch::Torch;
+
+llvm::Optional<Value>
+createOneDimTfIndices(PatternRewriter &rewriter, Operation *op,
+                      SmallVector<int64_t> indicesOneDimShape, int32_t dim,
+                      ArrayRef<int64_t> indexShape) {
+  int indexRank = indexShape.size();
```

Comment:
nit: you can use `unsigned`

Reply:
unsigned int indexRank = indexShape.size();

Reply:
Just `usnigned index..`, it implicitly means int.

Reply:
Done